README.md

```diff
@@ -7,51 +7,56 @@ Originally developed as [match4healthcare](https://github.com/match4everyone/mat
 
 ## Quick install
 - Copy `backend.prod.env.example` and `database.prod.env.example` to `backend.prod.env` and `database.prod.env` and fill in appropriate values
-- Run `./deploy.sh` (uses Docker) and visit `http://localhost:8000/`
+- Run `docker-compose -f docker-compose.yml -f docker-compose.prod.yml up -d --build` (uses Docker) and visit `http://localhost:8000/`
 
+## Environment variables
+
+TODO
```

Comment:
A miracle that @bjrne's pre-commit didn't spot this 😝 @maltezacharias Did you intend to document it in this PR?

Reply:
Nope, I just noticed we never updated that section and we have an outstanding issue for the Readme/Doc anyway. I felt lucky when it didn't catch me :D. I did add a reference to the sample .envs instead, and checked the documentation in there